<?xml version="1.0"?>
<launch>
  <arg name="chain_start" default="base_link" />
  <arg name="chain_end" default="moveo_nozzle_link" />
  <arg name="timeout" default="5" />
  <arg name="eps" default="1e-5"/>
  <arg name="model" default="$(find moveo_urdf)/urdf/moveo_urdf.urdf"/>
  <param name="robot_description" textfile="$(arg model)" />
  <arg name="joint_division" default="$(find gcode_translation)/include/Joint_division/Joint_division.csv"/>

  <node name="moveo_moveit_move_group" pkg="moveo_moveit_move_group" type="set_joint" output="screen">
    <param name="chain_start" value="$(arg chain_start)"/>
    <param name="chain_end" value="$(arg chain_end)"/>
    <param name="timeout" value="$(arg timeout)"/>
    <param name="eps" value="$(arg eps)" type = "double"/>
    <param name="urdf_param" value="/robot_description"/>
    <param name="joint_division" value="$(arg joint_division)"/>
  </node>
</launch>
